lib/usd/translators/shading/usdUVTextureReader.cpp

```diff
@@ -67,6 +67,9 @@ TF_DEFINE_PRIVATE_TOKENS(
     // UsdUVTexture Input Names
     (bias)(fallback)(scale)(wrapS)(wrapT)
 
+    // uv connections:
+    (outUvFilterSize)(uvFilterSize)(outUV)(uvCoord)
```

Comment:
Similar to the `clang-format` comment above, these would ideally be one per line.